samples/modules/lvgl/lvgl_input/README.rst

```diff
@@ -1,5 +1,5 @@
-.. zephyr:code-sample:: lvgl
-   :name: LVGL basic sample
+.. zephyr:code-sample:: lvgl_input
+   :name: basic LVGL input sample
```

Comment:
code sample identifier should be consistent with other LVGL samples + suggested name improvement (also dropping "sample" as it's to be avoided when using the code-sample directive)

```suggestion
.. zephyr:code-sample:: lvgl-input-devices
   :name: LVGL input devices
```